samples/matter/common/src/bt_nus_service.cpp

```diff
@@ -60,20 +60,25 @@ bool NUSService::Init(uint8_t priority, uint16_t minInterval, uint16_t maxInterv
 
 	mAdvertisingRequest.onStarted = [](int rc) {
 		if (rc == 0) {
-			LOG_INF("NUS BLE advertising started");
+			GetNUSService().mIsStarted = true;
```

Comment:
I think `GetNUSService()` is not needed, as the field should be accessible directly from this method.

Reply:
But then I cannot assign in the simple way that lambda to onStarted method.